http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/WsmDao.scala

```diff
@@ -86,6 +86,10 @@ trait WsmDao[F[_]] {
     ev: Ask[F, AppContext]
   ): F[Option[WorkspaceDescription]]
 
+  def getLandingZoneResources(billingProfileId: String, userToken: Authorization)(implicit
```

Comment:
It'd be nice to remove methods in WsmDAO that we aren't using anymore, like `createNetwork`.

Reply:
Would still be good to do this, don't think we should be using the ip or relayNamespace methods anymore?